src/chips/nrf51/Makefile.mk

```diff
@@ -1,7 +1,7 @@
 ARCH = cortex-m0
 
 RUSTC_FLAGS += -C opt-level=3 -Z no-landing-pads
-RUSTC_FLAGS += --target $(SRC_DIR)chips/$(CHIP)/makefile_target.json
+RUSTC_FLAGS += --target $(SRC_DIR)chips/$(CHIP)/target.json
```

Comment:
neither `makefile_target` nor `target` will work for both `Makefile` based compilation from root and Cargo based compilation without fixing the relative locations of the two crates because of the where the linker script is expected to be. In the transition period where we're still doing both (maybe not for long?) I suggest we just keep both copies.

It works on the SAM4L port because we don't have to compile directly an ELF with Rust, since we are not incorporating any native-C code. But that's not the case with the NRF51 because of the code in `crt1.c`.